ecommerce/courses/utils.py

```diff
@@ -168,3 +174,24 @@ def get_is_personalized_recommendation(course, request):
                 return recommendations['is_personalized_recommendation']
 
     return None
+
+
+def get_course_info_from_lms(site, course_id, cache_key):
+    """
+    Get course information from the LMS courses API.
+
+    Arguments:
+        site (Site): Site object containing Site Configuration data.
+        course_id (str): Course ID pattern.
+        cache_key (str): Cache key for given resource
+
+    Returns:
+        dict: Course information.
+    """
```

Comment:
Following the comment before, I think is better put the variable here:

```suggestion
    """
    ` LMS_COURSES_API_BASE_URL = '/api/courses/v1/courses/'`
```